src/util/sandbox/fd_sandbox_linux_private.h

```diff
@@ -0,0 +1,82 @@
+#ifndef HEADER_fd_src_util_sandbox_fd_sandbox_linux_h
+#define HEADER_fd_src_util_sandbox_fd_sandbox_linux_h
+
+/* In order to be able to sandbox itself, Firedancer has to be started
```

Comment:
I feel like this is just going to get out of sync, better in a `security-architecture.md` file and then point to that from here.

Reply:
I can remove it once we have security-architecture.md.